osidb/tests/test_endpoints.py

```diff
@@ -1127,10 +1127,13 @@ def test_flaw_delete(self, auth_client, test_api_uri):
         assert response.status_code == 200
 
         response = auth_client.delete(f"{test_api_uri}/flaws/{flaw.uuid}")
-        assert response.status_code == 204
+        # this HTTP method is not allowed until we leave Bugzilla and
+        # define the conditions under which a flaw can be deleted
+        assert response.status_code == 405
 
-        response = auth_client.get(f"{test_api_uri}/flaws/{flaw.uuid}")
-        assert response.status_code == 404
+        # TODO resurrect eventually
+        # response = auth_client.get(f"{test_api_uri}/flaws/{flaw.uuid}")
+        # assert response.status_code == 404
```

Comment:
it may be only a personal preference and thus may be ignored if you will, but  I would like to suggest maybe remove this code inside comments and open a task linking this code from any other commit, like:

https://github.com/RedHatProductSecurity/osidb/blob/de526048739ebeb72c56f17aceee1f8d7b91744a/osidb/tests/test_endpoints.py#L1132